src/web/sourceUtils.ts

```diff
@@ -0,0 +1,20 @@
+﻿import * as vscode from "vscode";
+
+export const getRequires = (root: string, source: string): vscode.Uri[] => {
+  let result = [];
+
+  let sourceLines = source.split("\n");
+  for (let line of sourceLines) {
+    let m = line.match(/require\(["']([.\/_\-!@#$%^{}\[\]`~A-Za-z0-9]+)["']\)/);

```

Comment:
We'll see if my brain cooperates tomorrow. I may try to see if I can make webpack run on the fly and generate a sourcemapped file to execute. Would be nice since it could also theoretically handle typescript with some tweaks.